packages/sns/src/converters/governance.converters.ts

```diff
@@ -84,3 +86,43 @@ export const toStopDissolvingNeuronRequest = ({
     },
   ],
 });
+
+export const toSetDissolveTimestampRequest = ({
```

Comment:
maybe not for this PR but is there any way to refactor this converter? spontaneously seems like all functions have something in common